pkgs/by-name/be/benchexec/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3
+, libseccomp
+, nixosTests
+}:
+python3.pkgs.buildPythonApplication rec {
```

Comment:
I don't think python packages like this one belong to `pkgs/by-name` just yet. It's a challenge for later.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#limitations

Reply:
OK, will move them.